.github/workflows/conventional-pr.yml

```diff
@@ -16,3 +16,5 @@ jobs:
       - uses: CondeNast/conventional-pull-request-action@v0.2.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          commitTitleMatch: "false"
```

Comment:
I configured the merge strategy to be squashing with the title matching the title of the PR so we can set this one to false.